.github/workflows/ci-cd.yml

```
@@ -0,0 +1,54 @@
+          name: production-files
+          path: ./build
+
+  deploy:
+    name: Deploy
+    needs: build
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/dev'
+
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v2
```

actionlint (1.7.4)

the runner of "actions/download-artifact@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)